package.json

```diff
@@ -23,52 +39,33 @@
       "npm run lint:fix"
     ]
   },
+  "dependencies": {
+    "@decathlon/vitamin-css": "file:packages/css"
+  },
   "devDependencies": {
     "@babel/core": "^7.12.10",
     "@storybook/addon-actions": "^6.1.11",
     "@storybook/addon-essentials": "^6.1.11",
     "@storybook/addon-links": "^6.1.11",
     "@storybook/html": "^6.1.11",
-    "autoprefixer": "^10.1.0",
     "babel-loader": "^8.2.2",
     "chokidar-cli": "^2.1.0",
     "cross-env": "^7.0.3",
     "eslint": "^7.15.0",
     "eslint-config-airbnb": "^18.2.1",
     "husky": "^4.3.6",
     "lerna": "^3.22.1",
-    "link-parent-bin": "^2.0.0",
     "lint-staged": "^10.5.3",
     "npm-run-all": "^4.1.5",
-    "postcss": "^8.2.1",
-    "postcss-cli": "^8.3.1",
-    "postcss-import": "^14.0.0",
+    "postcss-cli": "^7.1.2",
+    "postcss-import": "^12.0.0",
+    "postcss-loader": "^4.1.0",
     "prettier": "^2.2.1"
   },
-  "workspaces": [
-    "packages/*"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Decathlon/vitamin-web.git"
-  },
-  "bugs": {
-    "url": "https://github.com/Decathlon/vitamin-web/issues"
-  },
-  "keywords": [
-    "design-system",
-    "ui-kit",
-    "libraries"
-  ],
-  "license": "Apache-2.0",
   "licenses": [
```

Comment:
You can also move `licenses` to top as you did with `license` and other properties.